pkg/kubelet/dockertools/manager.go

```diff
@@ -1494,14 +1494,28 @@ func (dm *DockerManager) runContainerInPod(pod *api.Pod, container *api.Containe
 	if err != nil {
 		return kubecontainer.ContainerID{}, fmt.Errorf("InspectContainer: %v", err)
 	}
+
+	if !containerInfo.State.Running {
```

Comment:
I already fixed it in the latest commit.

Our current logic doesn't check if the pid is 1 or not. It only checks if it's 0. Why do we need to check for 1?